src/Dispatcher/WrappedDispatcher.php

```diff
@@ -24,8 +36,26 @@ public function dispatch(ServerRequestInterface $request, ResponseInterface $res
 		$request = $this->createApiRequest($request);
 		$response = $this->createApiResponse($response);
 
-		// Dispatch our classes
-		$response = $this->inner->dispatch($request, $response);
+		try {
+			// Dispatch our classes
+			$response = $this->inner->dispatch($request, $response);
+		} catch (Throwable $exception) {
+			// Log exception
```

Comment:
Error handling bych udelal spis tak, ze v catch metode se zavola neco jako `$this->errorHandler->handle()` a defaultni implementace by v sobe mela ten logger treba. A clovek by si to mohl pretizit podle sebe. Takhle to moc nemuze.

Reply:
Done. Ještě začlením Tracy, ať to generuje i bluescreeny.